pkg/cri/sbserver/container_create.go

```diff
@@ -57,6 +57,8 @@ func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateConta
 		return nil, fmt.Errorf("failed to find sandbox id %q: %w", r.GetPodSandboxId(), err)
 	}
 	sandboxID := sandbox.ID
+
+	// TODO: Add PID endpoint to Controller interface.
```

Comment:
Doesn't Status() expose pid as one of the fields returned? Do you mean add a new method that just returns the pid (if it's been started)?

Reply:
Yeah, I `Status` can cover both cases. No need in dedicated method for this.